### PR TITLE
Devhub: Dark mode

### PR DESCRIPTION
**Pull request by sentientwaffle**

### CODE REVIEW

**Comment by matklad** [comment]
😎

**Review by matklad — APPROVED** [review]

**Comment by sentientwaffle** [comment]
Force-pushed b/c github actions was broken (on [github's end](https://www.githubstatus.com/)) -- no actual code changes.

**Comment by sentientwaffle** [comment]
Huh, I expected to need a re-approval before it would merge due to the force-push.

**Pull request merged**
